packages/launcher/src/renderer/launcher/CompleteOnboardSession.js

```diff
@@ -0,0 +1,94 @@
+//@flow
+import React, { Component } from 'react'
+import styled from 'styled-components/native'
+import { Button, Text } from '@morpheus-ui/core'
+
+//import Icons
+import IdentityIcon from '@morpheus-ui/icons/IdentityMd'
+import IdentityFilledIcon from '@morpheus-ui/icons/IdentityFilledMd'
+import ContactsIcon from '@morpheus-ui/icons/ContactsMd'
+import ContactsFilledIcon from '@morpheus-ui/icons/ContactsFilledMd'
+import WalletsIcon from '@morpheus-ui/icons/WalletsMd'
+import WalletsFilledIcon from '@morpheus-ui/icons/WalletsFilledMd'
+import SkipIcon from '@morpheus-ui/icons/ArrowRightSm'
+
+const BUTTONS: Object = {
+  identities: {
+    title: 'Complete identity',
+    Icon: IdentityIcon,
+    HoverIcon: IdentityFilledIcon,
+  },
+  wallets: {
+    title: 'Setup your wallet',
+    Icon: WalletsIcon,
+    HoverIcon: WalletsFilledIcon,
+  },
+  contacts: {
+    title: 'Connect with others',
+    Icon: ContactsIcon,
+    HoverIcon: ContactsFilledIcon,
+  },
+}
+
+const BUTTONS_LIST = ['identities', 'wallets', 'contacts']
+
+const Container = styled.View``
+
+const ButtonsContainer = styled.View`
+  padding: 20px 0;
+  flex-direction: row;
+  align-items: center;
+  overflow-x: auto;
+`
+
+const ButtonWrapper = styled.View`
+  margin-right: 20px;
+`
+
+type Props = {
+  onSelectItem: (item: string) => void,
+  onSkip: () => void,
+}
+
+export default class CompleteOnboardSession extends Component<Props> {
+  render() {
+    return (
+      <Container>
```

Comment:
Could we use a React `Fragment` instead of an empty container here?

Reply:
good idea. I'll put it on the next PR